scripts/install

```diff
@@ -42,8 +42,10 @@ test ! -e "$final_path" || ynh_die --message="This path already contains a folde
 # Register (book) web path
 ynh_webpath_register --app=$app --domain=$domain --path_url=$path_url
 
-# Check memory requirements
-check_memory_requirements
+if [ ${PACKAGE_CHECK_EXEC:-0} -eq 0 ]; then
```

Comment:
Wouldn't it be better to have package_check set up a swap partition, in order to test that mechanism? Couldn't we also be stuck that way with some application refusing to start without that partition in the package_check environment...? 🤔

Reply:
Not sure the issue comes from package_check ... the package_check on my own CI works fine ... no memory issue

Reply:
I've removed it